chsdi/lib/validation/__init__.py

```diff
@@ -61,7 +61,7 @@ def srid(self):
 
     @srid.setter
     def srid(self, value):
-        if value in map(str, SUPPORTED_OUTPUT_SRS):
+        if value in list(map(str, SUPPORTED_OUTPUT_SRS)):
```

Comment:
`list` here is not needed and is bad. `map` in python3 returns an iterator that is more efficient than a list when used with a generator.